Code/Framework/AzNetworking/AzNetworking/Serialization/ISerializer.h

```diff
@@ -9,6 +9,7 @@
 #pragma once
 
 #include <stdint.h>
+#include <AzCore/base.h>
```

Comment:
NP: You could macro wrap this include as well if you wanted to

Reply:
Nice catch! Thank you :)

Reply:
Actually conditionally including this macro is not correct.

The `AZ_TRAIT_COMPILER_INT64_T_IS_LONG` comes from the `AzCore_Traits_Platform.h` that include that base.h brings in.

For example on Linux this macro is defined by the platform specific [AzCore_Traits_Linux.h](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Platform/Linux/AzCore/AzCore_Traits_Linux.h#L59).

Without the include, the macro is defined.

Reply:
Fixed! Thanks :)